src/hooks/useQuery.ts

```diff
@@ -13,6 +13,7 @@ export interface UseQueryArgs<V> {
   query: string | DocumentNode;
   variables?: V;
   requestPolicy?: RequestPolicy;
+  operationContext?: Partial<OperationContext>;
```

Comment:
I was actually thinking we could just make this a rest spread? Or name it `context` at least, I suppose? I'm not quite sure.

What do you think, @andyrichardson?

Also, you're just too fast, Jovi 😆

Reply:
The word _context_ in our client is just confusing when working with React 😄 

Prefixing it with operation makes sense but I'm happy with either.

Reply:
I understand what @andyrichardson is saying, context could be confusing another option would be to use the word `defaultOperationContext` but that could be quite long ^^